src/tests/MovieDetails.test.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import MovieDetails from '../components/MovieDetails';
+
+describe('MovieDetails', () => {
+  const movieData = {
+    poster_path: 'test-poster.jpg',
+    title: 'Test Movie',
+    vote_average: 8.5,
+    genres: ['Action', 'Adventure'],
+    release_date: '2020-01-01',
+    runtime: 120,
+    overview: 'Test overview',
+  };
+
+  it('renders movie image', () => {
```

Comment:
Please change also in other places 
```suggestion
  it('should render movie image', () => {
```